### ISSUE TITLE
Build errors on Xcode 9 beta 4 with Swift 3.2

### ISSUE THREAD

**Issue by ldiqual** [nvzqz/RandomKit#39]
![image](https://user-images.githubusercontent.com/941453/28727342-5e01b2ae-7379-11e7-9b48-68b55221438a.png)

```
RandomKit/Extensions/Swift/Strideable+RandomKit.swift:48:1: Type 'String.UTF16Index' (aka 'String.Index') does not conform to protocol 'RandomInRange'

RandomKit/Extensions/Swift/Strideable+RandomKit.swift:48:1: Type 'String.UTF16Index' (aka 'String.Index') does not conform to protocol 'RandomInClosedRange'

RandomKit/Extensions/Swift/Strideable+RandomKit.swift:48:1: Type 'String.UTF16Index' (aka 'String.Index') does not conform to protocol 'RandomInRange'
```

via cocoapods

```
pod 'RandomKit', '5.2.0'
```

**Comment by nvzqz** [comment]
This may be due to recent changes in Swift 3.2 & 4 regarding `String` indexing and `String` now being a collection. Thanks for pointing this out; I'll try to fix this soon.

**Comment by nvzqz** [comment]
Just released [v5.2.1](https://github.com/nvzqz/RandomKit/releases/tag/v5.2.1) with this change.

**Comment by nvzqz** [comment]
Just released [v5.2.2](https://github.com/nvzqz/RandomKit/releases/tag/v5.2.2) which reimplements this feature that was removed in [v5.2.1](https://github.com/nvzqz/RandomKit/releases/tag/v5.2.2).
